### PR TITLE
documentation(EJ2-65674): UG documentation for Mention integration with Rich Text Editor

### PR DESCRIPTION
**Pull request by Kokila13081999**
documentation(EJ2-65674): UG documentation for Mention integration with Rich Text Editor

### CODE REVIEW

**Comment by Kokila13081999** [comment]
rebuild

File: ej2-asp-core-mvc/code-snippet/rich-text-editor/mention-integration/SportsData.cs [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    {
+        public string Name { get; set; }
+        public string EmployeeImage { get; set; }
+        public string EmailId { get; set; }
+
+        public List<SportsData> SportsList()
+        {
+            List<SportsData> mention = new List<SportsData>();
+            mention.Add(new SportsData { Name = "Selma Rose", EmployeeImage = "3.png", EmailId = "selma@gmail.com" });
+            mention.Add(new SportsData { Name = "Russo Kay", EmployeeImage = "8.png", EmailId = "russo@gmail.com" });
```
Comment (thangavele) [inline ej2-asp-core-mvc/code-snippet/rich-text-editor/mention-integration/SportsData.cs:18]: image


is it sports data? Why you are repeating this type of issue?

Comment (Kokila13081999) [reply ej2-asp-core-mvc/code-snippet/rich-text-editor/mention-integration/SportsData.cs]: Resolved

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/mention-integration.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Mention Integration in ##Platform_Name## Rich Text Editor | Syncfusion
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/mention-integration.md:3]: You can use the direct platform name. don't use the template

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/mention-integration.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+title: Mention Integration in ##Platform_Name## Rich Text Editor | Syncfusion
+description: Learn here all about Mention Integration in Syncfusion ##Platform_Name## Rich Text Editor control of Syncfusion Essential JS 2 and more.
```
Comment (thangavele) [inline ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/mention-integration.md:4]: ##Platform_Name##   Dont use this template. You can directly specify the platform

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/mention-integration.md (hunk 2 above)
Comment (Kokila13081999) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.MVC/mention-integration.md]: Corrected

File: ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/mention-integration.md (hunk 3 above)
Comment (Kokila13081999) [reply ej2-asp-core-mvc/rich-text-editor/EJ2_ASP.NETCORE/mention-integration.md]: corrected

**Review by thangavele — APPROVED** [review]

**Pull request merged**